xml/cl.xml

```diff
@@ -1113,7 +1125,14 @@ server's OpenCL/api-docs repository.
         <enum value="0x105B"        name="CL_DEVICE_IL_VERSION_KHR"/>
         <enum value="0x105C"        name="CL_DEVICE_MAX_NUM_SUB_GROUPS"/>
         <enum value="0x105D"        name="CL_DEVICE_SUB_GROUP_INDEPENDENT_FORWARD_PROGRESS"/>
-            <unused start="0x105E" end="0x107F" comment="Reserved for cl_device_info"/>
+        <enum value="0x1063"        name="CL_DEVICE_ATOMIC_MEMORY_CAPABILITIES"/>
```

Comment:
Are `0x105E` through `0x1062` going to be used for something else?

Reply:
Yes, they are used for the extended versioning extension, see #218 .

Thanks for the review!

Reply:
Okay, got it.